support.js

```diff
@@ -303,8 +304,9 @@ if (async.length) {
 }
 
 function onLoaded(fn) {
-  if (document.readyState == "loading") {
-    document.addEventListener('DOMContentLoaded', fn);
+  if (document.readyState == "loading" ||
+      document.readyState == "interactive") {
+    window.addEventListener('load', fn);
```

Comment:
It does, but it fires when the document enters the 'interactive' state, at which point `document.body` is still `null`.

I think this isn't ideal on a real page, since it waits for images, etc, to load, but since it's just the support page (and has no external resources) it seems best to keep it simple.